docs/how_to_guides/how_to_scale_chemical_reactions.rst

```diff
@@ -0,0 +1,158 @@
+.. _how_to_scale_chemical_reactions:
+
+How to scale chemical reactions
+===============================
+
+.. warning::
+    DO NOT use 'user-scaling' as a solver option when scaling any chemistry module.
```

Comment:
Should we rephrase this to sound less absolute? Perhaps something along the lines of 'user-scaling' not currently supported, recommending not to use this, and for users to try at their own risk.

Furthermore, a note on what the default solver option is, which is sufficient for chemistry modules (for those who might read this without knowing anything about the solver options).

Reply:
I feel like this would need to be its own documentation. It would be aligned with the initialization and solve steps of a chemistry module, since that is where you actually invoke the solver and set solver options. I put the warning here because this is directly related to scaling, but solver options should be their own documentation, but maybe link those documents to each other?

Reply:
A link between those docs would be great, but I think for now, just removing the words "DO NOT" to something more flexible like "It is not recommended to" would suffice for this PR. If you go beyond that (e.g., providing a link to documents related to solver options, etc. ) even better.